part-spec/power.json

```diff
@@ -697,6 +697,507 @@
+        },
+        "fswMin": {
+          "description": "minimum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "fswMax": {
+          "description": "maximum switching frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentConsumption": {
+          "description": "current consumption of a device",
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/currentConsumption"
+          }
+        },
+        "integratedFets": {
+          "description": "whether a device contains integrated switching mosfets",
+          "type": "boolean"
+        },
+        "integratedFetProperties": {
+          "description": "describes integrated fet current limits and rdson properties",
+          "$ref": "#/$defs/powerFetProperities"
+        },
+        "currentMatchingAccuracyTyp": {
+          "description": "typical current matching between LED strings",
+          "comment": "units of percentage",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentMatchingAccuracyMin": {
+          "description": "minimum current matching between LED strings",
+          "comment": "units of percentage",
+          "$ref": "definitions.json#/unit"
+        },
+        "currentMatchingAccuracyMax": {
+          "description": "maximum current matching between LED strings",
+          "comment": "units of percentage",
+          "$ref": "definitions.json#/unit"
+        },
+        "integratedBoost": {
+          "description": "whether a boost switching converter is integrated in the device",
+          "type": "boolean"
+        },
+        "numberLEDStrings": {
+          "description": "maximum number of LED strings a device can support",
+          "type": "number"
+        },
+        "numberLEDPerString": {
+          "description": "maximum number of LEDs per string a device can support",
+          "type": "number"
+        },
+        "dimmingSupport": {
+          "description": "whether a device supports output current dimming",
+          "type": "boolean"
+        },
+        "dimmingFrequencyTyp": {
+          "description": "typical dimming frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "dimmingFrequencyMin": {
+          "description": "minimum dimming frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "dimmingFrequencyMax": {
+          "description": "maximum dimming frequency of a device",
+          "comment": "units of hertz",
+          "$ref": "definitions.json#/unit"
+        },
+        "dimmingRatioTyp": {
+          "description": "typical dimming ratio of a device",
+          "type": "string",
+          "example": "20:1"
```

Comment:
Ideally, tools wouldn't have to parse strings for structured information. Should we have something like a "Ratio" type which has a numerator and denominator?

Reply:
Good idea. I'll work on some proposals to update the definition file and this property.